src/main/kotlin/racing/ui/ResultView.kt

```diff
@@ -8,22 +11,36 @@ object ResultView {
 
     private const val SYMBOL_ONE_STEP = "-"
     private const val NOTICE_RESULT_START = "실행결과"
+    private const val NAME_DELIMETER_FOR_DISPLAY = ", "
+    private const val EXCEPTION_MESSAGE_EMPTY_LIST = "리스트의 값이 존재하지 않습니다."
 
     fun printResultNotice() {
         println(NOTICE_RESULT_START)
     }
 
-    fun printAllPosition(positions: List<Int>) {
-        for (position in positions) {
-            printSinglePosition(position)
+    fun printRacingHistory(racingHistory: RacingHistory) {
+        val racingResults = racingHistory.roundResults
+        racingResults.forEach { printRoundResult(it) }
+    }
+
+    private fun printRoundResult(roundResult: RoundResult) {
+        val carStates = roundResult.carStates
+        carStates.forEach { (carName, carPosition) ->
+            printCarStates(carName.name, carPosition.position)
         }
         println()
     }
 
-    private fun printSinglePosition(position: Int) {
+    private fun printCarStates(name: String, position: Int) {
+        print("$name : ")
```

Comment:
```suggestion
      println("$name : ${SYMBOL_ONE_STEP.repeat(position)}")
```

위와 같이 바꿀 수 있을 것 같네요!

Reply:
와 Kotlin에서 문자열 활용이 어마무시하네요..!